### ISSUE TITLE
Example of rule is unvalid

### ISSUE THREAD

**Issue by hyphaene** [kristerkari/stylelint-z-index-value-constraint#2]
Hi, 

I wanted to let you know that the given example does not present a valid one. 
```json
{
  "plugins": ["stylelint-z-index-value-constraint"],
  "rules": [
    "plugin/z-index-value-constraint": {
      "min": 1,
      "max": 10
    }
  ]
}
```

(as the red color indicates it) 

I can make a PR for this is you want.

Thx !

**Comment by kristerkari** [comment]
Thanks for this @hyphaene! Yes, the `rules` key should be an object instead of an array. 

I totally missed that! :)

**Comment by kristerkari** [comment]
Fixed in 38d3cea2d3c64123f1fa7df7bf5227b3111b0346

Thanks, and let me know if you find something else that is not working!

**Comment by hyphaene** [comment]
Sure no problem ! Thx for the quick reaction ;)
